include/GoogleSync/GoogleSync.php

```diff
@@ -1,4 +1,5 @@
 <?php
```

Comment:
Test Coverage is only 37%, needs more test.

Reply:
Yep. I'm going to try to write some more tests. What's going to trip me up is writing tests for methods that require communication with google, and a valid sync config set up.